de.tubs.variantsync.core/src/de/tubs/variantsync/core/VariantSyncComposer.java

```diff
@@ -17,93 +17,73 @@
 
 public class VariantSyncComposer extends ComposerExtensionClass {
 
-	@Override
-	public void performFullBuild(Path config) {
-//		public void performFullBuild(IFile config) {
+	@Override public void performFullBuild(Path config) {
 		VariantSyncPlugin.getConfigurationProjectManager().reinitialize();
 	}
 
-	@Override
-	public void addCompiler(IProject project, String sourcePath, String configPath, String buildPath) {
+	@Override public void addCompiler(IProject project, String sourcePath, String configPath, String buildPath) {
 		File featureContextFile = new File(project.getFile(FeatureContextFormat.FILENAME).getLocationURI());
 		try {
 			featureContextFile.createNewFile();
 		} catch (IOException e) {
-			// TODO Auto-generated catch block
 			e.printStackTrace();
 		}
 	}
 
-	@Override
-	public boolean hasSourceFolder() {
+	@Override public boolean hasSourceFolder() {
```

Comment:
Why is the `@Override` inline?

Reply:
This is  the output if the provided code style is used (with automated code  formatting)